CHANGELOG/CHANGELOG-1.18.md

```diff
@@ -719,7 +719,9 @@ filename | sha512 hash
 ### Bug or Regression
 
 - "unbound immediate PersistentVolumeClaims" causes UnschedulableAndUnresolvable status rather than an Error in the scheduler. ([#93892](https://github.com/kubernetes/kubernetes/pull/93892), [@ahg-g](https://github.com/ahg-g)) [SIG Apps and Storage]
-- $ kubectl get event
+- kubectl: fix the Scheduled eventTime is \<unknown\> when use kubectl describe pod xxx. ([#94226](https://github.com/kubernetes/kubernetes/pull/94226), [@ingvagabund](https://github.com/ingvagabund)) [SIG CLI]
+  ```sh
+  $ kubectl get event
```

Comment:
The appropriate fix for this is to look at the "Does this PR introduce a user-facing change?" on the associated PR (#94226). Since it's a backport, sometimes the release note doesn't get copied over from the original PR so look at the original PR (#90227) and you'll find the right release notes which should be in this place.

Reply:
Thanks for your reviews @wilsonehusin , fixed.